Lib/test/test_io.py

```diff
@@ -1726,6 +1737,16 @@ def bad_write(b):
         self.assertRaises(OSError, b.close) # exception not swallowed
         self.assertTrue(b.closed)
 
+    def test_slow_close_from_thread(self):
+        rawio = self.SlowFlushRawIO()
```

Comment:
Can you add a comment pointing to the bug entry here?

Reply:
It looks like you forgot to address this comment.

Reply:
Ah, whoops. Thank you.